cron_jobs/cronDemarchesSimplifiees.ts

```diff
@@ -65,11 +67,11 @@ const checkForMultipleAcceptedDossiers = async (): Promise<boolean> => {
 };
 
 const autoAcceptPsychologists = async (): Promise<void> => {
-  demarchesSimplifiees.autoAcceptPsychologist();
+  autoAccept.autoAcceptPsychologist();
```

Comment:
Since these files (autoAccept and autoVerify) only export one method, we could call it directly here to avoid redundancy.

Reply:
because i don't know how to properly stub this if it's a default export :(